packages/react-hooks/src/MedplumProvider/MedplumProvider.test.tsx

```diff
@@ -402,8 +402,8 @@ describe('MedplumProvider', () => {
       expect(screen.getByText('Loading...')).toBeInTheDocument();
       expect(medplum.isLoading()).toEqual(true);
 
-      await expect(screen.findByText('Parent count: 2')).resolves.toBeInTheDocument();
-      expect(screen.getByText('Child count: 1')).toBeInTheDocument();
+      await expect(screen.findByText('Child count: 1')).resolves.toBeInTheDocument();
```

Comment:
Need to check child first since it's dependent on an async change that is dependent on parent change (this order ensures both conditions are met)